tests/tt_metal/tt_metal/unit_tests_fast_dispatch/common/command_queue_fixture.hpp

```diff
@@ -4,9 +4,6 @@
 
 #include "gtest/gtest.h"
 #include "tt_metal/detail/tt_metal.hpp"
-#include "tt_metal/host_api.hpp"
-#include "tt_metal/impl/dispatch/command_queue.hpp"
-#include "tt_metal/llrt/rtoptions.hpp"
 #include "tt_metal/test_utils/env_vars.hpp"
 
 using namespace tt::tt_metal;
```

Comment:
😢

Reply:
Maybe it is a wrong place but it would be nice to remove using namespace from headers.